executor/oci/spec_unix.go

```diff
@@ -44,6 +55,10 @@ func GenerateSpec(ctx context.Context, meta executor.Meta, mounts []executor.Mou
 	s.Process.Env = meta.Env
 	s.Process.Cwd = meta.Cwd
 
+	if !hasEnv("PATH", s.Process.Env) {
+		s.Process.Env = append(s.Process.Env, "PATH="+system.DefaultPathEnv)
+	}
```

Comment:
This could be achieved above with `s.Process.Env = append(s.Process.Env, meta.Env...)` instead since the containerd `GenerateSpec` also sets a default path.

I decided since there were other uses of `system.DefaultPathEnv` it was more internally consistent this way around.